README.md

```diff
@@ -20,6 +20,7 @@
     * [Sending metrics to AWS IoT](#sending-metrics-to-aws-iot)
 * [Versioning](#versioning)
 * [Releases and Documentation](#releases-and-documentation)
+    * [202211.00](#20210800)
```

Comment:
Is the MQTT porting guide updated (especially with the DQP and LTS updates)?

Reply:
Yeah, Jason wrote the porting guide when doing the update:
https://github.com/FreeRTOS/coreMQTT/blob/main/MigrationGuide.md